src/ontology/cl-edit.owl

```diff
@@ -32543,6 +32544,17 @@ AnnotationAssertion(terms:date obo:CL_4047012 "2024-09-06T08:41:39Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047012 "angiogenic pericyte"@en)
 SubClassOf(obo:CL_4047012 obo:CL_0000669)
 
+# Class: obo:CL_4047017 (transit amplifying cell of gut)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20683682") Annotation(oboInOwl:hasDbXref "PMID:33195268") Annotation(oboInOwl:hasDbXref "PMID:34497389") obo:IAO_0000115 obo:CL_4047017 "A transit amplifying cell of the gut is located in the intestinal crypt above the stem cell zone. This cell is part of a rapidly dividing cell population derived from intestinal stem cells. It is capable of multiple rounds of division before terminal differentiation. This cell gives rise to all differentiated intestinal epithelial cell types including absorptive enterocytes and secretory cells (goblet, enteroendocrine, and paneth cells). A transit amplifying cell of the gut expresses markers such as MCM5, TOP2A, CCNA2, PCLAF, HELLS, PCNA, and MKI67 in humans.")
```

Comment:
"A transit amplifying cell of the gut is located in the intestinal crypt above the stem cell zone. This cell is part of a rapidly dividing cell population derived from intestinal stem cells. It is capable of multiple rounds of division before terminal differentiation. This cell gives rise to all differentiated intestinal epithelial cell types including absorptive enterocytes and secretory cells (goblet, enteroendocrine, and paneth cells). A transit amplifying cell of the gut expresses markers such as MCM5, TOP2A, CCNA2, PCLAF, HELLS, PCNA, and MKI67 in humans.")"

Standard style is for first sentence to start with name of the parent/general cell type rather than to name the type being defined.  Could also be s bit more precise with location.  Suggested revision:

"A transit amplifying cell of the gut epithelium, located in the wall of the intestinal crypt, just above intestinal stem cells from which they derive. These are rapidly dividing cells, capable of multiple rounds of division before differentiating into the various cell types of the gut epithelium (enterocyte, goblet, eneterodendocrine, paneth cells)."

Re markers - this seems to be the list of markers from the GCA spreadsheet.  Not all are supported by the refs.  I think the GCA marker lists are not yet stable enough to fold into CL defs.  So, I think leave out for now.

Also - we can add a relationship:

develops_directly_from some 'intestinal epithelium stem cell'

Reply:
Amended def to suggested. logical def has been updated to 'directly develops from' some 'intestinal crypt stem cell' as 'develops directly from' is not an option  and 'intestinal epithelium stem cell' is not present in the ontology but is synonymous with 'intestinal crypt stem cell'